### PR TITLE
finished lab

### PR DESCRIPTION
**Pull request by Yadadye**

### CODE REVIEW

**Comment by khatchad** [comment]
I don't have a Blackboard submission from your group. Please note the submission instructions and submit the GitHub username of the account you'd like me to grade ASAP:

> ... 
> 1. [Create a **pull request**](https://help.github.com/articles/creating-a-pull-request) on the original repository to turn in the assignment.
> 2. Submit your GitHub **username** on [Blackboard](https://bbhosted.cuny.edu/webapps/assignment/uploadAssignment?content_id=_20246794_1&course_id=_1101235_1&assign_group_id=&mode=cpview).

**Comment by Yadadye** [comment]
Done.

File: Library/src/Library.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+public class Library {
```
Comment (khatchad) [inline Library/src/Library.java:1]: :-1: Point reduction for invalid source code formatting.

File: Library/src/Library.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+// TODO: Add the missing implementation to this class
+ String address;
+ java.util.ArrayList<Book> books;
+
+ public static final String openTime = "9am";
```
Comment (khatchad) [inline Library/src/Library.java:6]: `OPEN_TIME` likewise below.

File: Library/src/lab1/Book.java [hunk 3]
```diff
@@ -1,67 +1,47 @@
...
+ return borrowed;
+}
+/**
+@@ -52,16 +57,17 @@ public boolean isBorrowed() {
+*/
+public String getTitle() {
+// TODO: Implement this method
+ return title;
+}
+public static void main(String[] arguments) {
```
Comment (khatchad) [inline Library/src/lab1/Book.java:34]: :-1: Per the instructions:

> You must not modify the main method

File: Library/src/Library.java (hunk 1 above)
Comment (khatchad) [inline Library/src/Library.java:1]: :-1: You added a library class? Not to spec.

**Comment by khatchad** [comment]
:-1: Doesn't compile.

**Pull request closed without merging**